devel/onetbb/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        oneapi-src oneTBB 2021.5.0 v
+name                onetbb
+revision            0
+categories          devel parallel
+platforms           darwin
+supported_archs     i386 x86_64
+license             Apache-2
+maintainers         {outlook.com:mohd.akram @mohd-akram} openmaintainer
+
+description         oneAPI Threading Building Blocks
+
+long_description    oneTBB is a flexible C++ library that simplifies the work \
+                    of adding parallelism to complex applications, even if you \
+                    are not a threading expert.
+
+checksums           rmd160  987a6f3ae0ce22cc00e21009c65c07ebe0e9cedb \
+                    sha256  be339d0c19815b4ab577f0dd51cd0d2d6ebbaf3b57756f52f9cc84eb85560834 \
+                    size    2463469
+
+depends_lib-append  port:hwloc
+
+compiler.cxx_standard  2011
+
+configure.args-append  -DTBB_TEST=OFF
```

Comment:
perhaps we should build the examples (`-DTBB_EXAMPLES=ON`)?

Reply:
Seems out of scope for the package tbh, and there are quite a few examples.

Reply:
we do build the examples for the `tbb` port as well and it's part of the package, so why out-of-scope?

Reply:
I guess you mean copying the example source files not building them? It seems that's what `tbb` is doing.

Reply:
I mean adding `-DTBB_EXAMPLES=ON` as `configure.args` and let the build-system to whatever it wants/needs to do for that...

Reply:
Tried it. It wastes time building them but doesn't use them for anything.

Reply:
thanks - okay, let's leave them out for now then and we can revisit this later if needed